tests/test_hdl_dsl.py

```diff
@@ -627,8 +627,8 @@ def test_FSM_basic(self):
             "(sig a)": "comb",
             "(sig fsm_state)": "sync",
             "(sig b)": "sync",
-            "(sig fsm_ongoing_FIRST)": "comb",
-            "(sig fsm_ongoing_SECOND)": "comb",
+            "(sig)": "comb",
+            "(sig)": "comb",
```

Comment:
this ... does not really work this way (it's a dict, so the entries are deduplicated)

I'm not sure how to deal with this properly; perhaps this should just be converted to a NIR-based test or something...

Reply:
Overlooked in a find and replace. I was more curious about the FSM tests here.

Reply:
I fixed the duplicate entry.